chain/db/db.go

```diff
@@ -78,9 +79,9 @@ func NewDB(dialect, path string, metadata map[string]string) (*BandDB, error) {
 		"RESTRICT",
 	)
 
-	db.Model(&OracleScriptRevision{}).AddForeignKey(
-		"tx_hash",
-		"transactions(tx_hash)",
+	db.Model(&OracleScript{}).AddForeignKey(
```

Comment:
We should not remove OracleScriptRevision foreign key. Please double check your code before pushing for reviews.